package.json

```diff
@@ -48,6 +48,7 @@
     "lodash": "^4.17.14",
     "node-gyp": "^11.2.0",
```

Comment:
Looks like `node-gyp` released v12 yesterday and they updated themselves to `tar@7.5.2`. v12 doesn't look like a breaking change for us so it might be worth trying to upgrade that instead? https://github.com/nodejs/node-gyp/releases/tag/v12.0.0

Reply:
I'll give this a shot!

Reply:
tests definitely would have failed since they use node-gyp